### PR TITLE
Added Persian localization

### PR DESCRIPTION
**Pull request by armineslami**

### CODE REVIEW

**Comment by ranterle** [comment]
Hi @armineslami - thanks so much for the contribution - as previously mentioned by @msftclas, we need you to sign a contriubtion license agreement before we can accept the PR - it's quick and painless. Appreciate your efforts!

**Comment by ranterle** [comment]
Re-pinging @armineslami - please consider signing the CLA - it's really quick, otherwise I'm afraid we won't be able to accept the PR, sorry.

**Comment by armineslami** [comment]
Hi @ranterle, Sorry for late response i just did what you asked and signed for CLA.

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

File: hockeysdk/src/main/res/values-fa/strings.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      <string name="hockeyapp_feedback_email_hint">ایمیل</string>
+      <string name="hockeyapp_feedback_subject_hint">موضوع</string>
+      <string name="hockeyapp_feedback_message_hint">متن پیام</string>
+      <string name="hockeyapp_feedback_last_updated_text">آخرین بروز رسانی : %s</string>
+      <string name="hockeyapp_feedback_attachment_button_text">افزودن ضمیمه</string>
+      <string name="hockeyapp_feedback_send_button_text">ارسال نظر</string>
+      <string name="hockeyapp_feedback_response_button_text">افزودن پاسخ</string>
+      <string name="hockeyapp_feedback_refresh_button_text">تازه کردن</string>
+      <string name="hockeyapp_feedback_title">بازخورد</string>
+      <string name="hockeyapp_feedback_send_generic_error">پیام نمی تواند ارسال شود. لطفا مقادیر ورودی و اتصال خود به اینترنت را چک کرده و دوباره امتحان کنید.</string>
```
Comment (MehrdadMzfr) [inline hockeysdk/src/main/res/values-fa/strings.xml]: +      <string name="hockeyapp_feedback_send_generic_error">پیام نمی تواند ارسال شود. لطفا اطلاعات ورودی و اتصال خود به اینترنت را چک کرده و دوباره امتحان کنید.</string>

File: hockeysdk/src/main/res/values-fa/strings.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      <string name="hockeyapp_feedback_validate_email_empty">لطفا یک ایمیل وارد کنید</string>
+      <string name="hockeyapp_feedback_validate_name_error">لطفا نام خود را وارد کنید</string>
+      <string name="hockeyapp_feedback_validate_text_error">پیام نمی تواند ارسال شود. لطفا متن پیام خود را وارد کنید</string>
+      <string name="hockeyapp_feedback_generic_error">بروز خطا</string>
+      <string name="hockeyapp_feedback_attach_file">ضمیمه فایل</string>
+      <string name="hockeyapp_feedback_attach_picture">ضمیمه عکس</string>
+      <string name="hockeyapp_feedback_select_file">انتخاب فایل</string>
+      <string name="hockeyapp_feedback_select_picture">انتخاب عکس</string>
+      <string name="hockeyapp_feedback_max_attachments_allowed">تنها %s ضمیمه مجاز است.</string>
+      <string name="hockeyapp_feedback_attachment_loading">در حال بارگذاری…</string>
```
Comment (MehrdadMzfr) [inline hockeysdk/src/main/res/values-fa/strings.xml]: <string name="hockeyapp_feedback_attachment_loading">...در حال بارگذاری</string>

File: hockeysdk/src/main/res/values-fa/strings.xml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      <string name="hockeyapp_feedback_attachment_error">خطا</string>
+
+      <!-- Send Feedback Task -->
+      <string name="hockeyapp_feedback_sending_feedback_text">در حال ارسال بازخورد…</string>
```
Comment (MehrdadMzfr) [inline hockeysdk/src/main/res/values-fa/strings.xml]: move the ... to the left of the sentence

File: hockeysdk/src/main/res/values-fa/strings.xml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      <!-- Login Activity -->
+      <string name="hockeyapp_login_headline_text">برای دسترسی به این نرم افزار لطفا ایمیل و کلمه عبور حساب کاربری خود در HockeyApp را وارد کنید.</string>
+      <string name="hockeyapp_login_headline_text_email_only">لطفا برای دسترسی به این نرم افزار، ایمیل حساب کاربری خود در HockeyApp را وارد کنید.</string>
+      <string name="hockeyapp_login_missing_credentials_toast">لطفا گزینه های خالی را وارد کنید</string>
+      <string name="hockeyapp_login_email_hint">ایمیل</string>
+      <string name="hockeyapp_login_password_hint">کلمه عبور</string>
+      <string name="hockeyapp_login_login_button_text">ورود</string>
+
+      <!-- Paint Activity -->
+      <string name="hockeyapp_paint_indicator_toast">چیزی را رسم کنید!</string>
```
Comment (MehrdadMzfr) [inline hockeysdk/src/main/res/values-fa/strings.xml]: <string name="hockeyapp_paint_indicator_toast">!چیزی را رسم کنید</string>

**Comment by TroubleMakerBen** [comment]
Hey @armineslami,

could you address @MehrdadMzfr 's comments, please?

I'm happy to merge afterwards.

Best,
Benny

**Pull request merged**